src/containers/LogOutPage.js

```diff
@@ -0,0 +1,22 @@
+import { PropTypes, Component } from 'react';
```

Comment:
I think that the way to go with this is:  remove this file, create a logout action that basically does what you're doing on the `componentWillMount` function, and make the logout link/button to trigger that action.